web/src/unread.ts

```diff
@@ -964,6 +964,17 @@ export function get_msg_ids_for_topic(stream_id: number, topic_name: string): nu
     return unread_topic_counter.get_msg_ids_for_topic(stream_id, topic_name);
 }
 
+export function dms_has_mention(msg_ids: number | number[]): boolean {
+    const idsToCheck = Array.isArray(msg_ids) ? msg_ids : [msg_ids];
```

Comment:
This variable name doesn't follow our snake case coding style guidelines.

Reply:
Also I think this function signature should probably just accept an array of `message_ids`